### DISCUSSION TITLE
How to control the zoom speed?

### DISCUSSION THREAD

**Discussion by zxdawn** [opengeos/leafmap#468] [Q&A] [answered]
Is it possible to control the zoom speed? Let's say we have one interested location and wanna zoom out a little bit to check the surrounding region, but the default zoom setting leads to a much larger area.

**Comment by giswqs** [comment, marked as answer]
This appears to be an ipyleaflet issue. I thought it should work, but it doesn't. 

- https://github.com/jupyter-widgets/ipyleaflet/pull/608
- https://github.com/jupyter-widgets/ipyleaflet/issues/591

```python
import ipyleaflet
m = ipyleaflet.Map(center=[40, -100], zoom=4, zoom_delta=0.1)
m
```

**Reply by zxdawn** [reply]
Thank you for checking. CC @davidbrochart now.

**Reply by zxdawn** [reply]
I realize we need to use `zoom_delta=0.1` and `zoom_snap=0` together. However, it only has effects on clicking the zoom tool, not for the scroll tool.

**Reply by giswqs** [reply]
I see. It is better than nothing
